ui/views.py

```diff
@@ -227,6 +227,10 @@ def dispatch(self, *args, **kwargs):
     def extra_context(self):
         """Add to the context."""
         context = super(RepositoryView, self).extra_context()
+        vocabularies = [
+            x.replace(" ", "_")
```

Comment:
Yep. Do you have a suggestion? How about adding a method to the `Vocabulary` model such as `get_slug` instead of using the `name` attribute?